src/Avalonia.Controls/Templates/FuncTreeDataTemplate.cs

```diff
@@ -58,7 +58,7 @@ public class FuncTreeDataTemplate : FuncDataTemplate, ITreeDataTemplate
         /// <returns>The child items, or null if no child items.</returns>
         public InstancedBinding ItemsSelector(object item)
         {
-            return InstancedBinding.OneTime(this?._itemsSelector(item));
```

Comment:
I wonder what's the reason of such weird piece of code was.

Reply:
I guess I must have just got off working with js ;) But to be honest, no idea - either a brainfart or copy/replace gone wrong?